<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpWindow</class>
 <widget class="QDialog" name="HelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Support</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>4</number>
     </property>
     <item row="2" column="0" colspan="3">
      <widget class="QCheckBox" name="cbLogSipMsg">
       <property name="text">
        <string>Log SIP messages</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QPlainTextEdit" name="logBrowser">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="maximumBlockCount">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visit &lt;a href=&quot;http://taki.sourceforge.net/&quot;&gt;Taki website&lt;/a&gt; for documentation, latest Taki news, updates, and support. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frBBControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="6">
       <widget class="QPushButton" name="btClearBB">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bbFrameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="btCopyBB">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btCancelBB">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btEmailBB">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frDesktopControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QPushButton" name="btCopy">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btClear</sender>
   <signal>clicked()</signal>
   <receiver>logBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCopy</sender>
   <signal>pressed()</signal>
   <receiver>logBrowser</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCopy</sender>
   <signal>released()</signal>
   <receiver>logBrowser</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>pressed()</signal>
   <receiver>HelpWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbLogSipMsg</sender>
   <signal>stateChanged(int)</signal>
   <receiver>HelpWindow</receiver>
   <slot>logMsgChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancelBB</sender>
   <signal>clicked()</signal>
   <receiver>HelpWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btClearBB</sender>
   <signal>clicked()</signal>
   <receiver>logBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCopyBB</sender>
   <signal>pressed()</signal>
   <receiver>logBrowser</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCopyBB</sender>
   <signal>released()</signal>
   <receiver>logBrowser</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEmailBB</sender>
   <signal>clicked()</signal>
   <receiver>HelpWindow</receiver>
   <slot>emailLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>logMsgChanged(int)</slot>
  <slot>emailLog()</slot>
 </slots>
</ui>
